UPP update fails to compile in UFS with type mismatch

In trying to update UPP hash to top of develop in ufs-weather-model, UFS RTs are bit-for-bit except for 24 applications that fail to compile.

The compile failures share these errors:
```
/scratch3/NCEPDEV/nems/Nick.Szapiro/tasks/warnings_upp/ufs-weather-model/UFSATM/upp/sorc/ncep_post.fd/CALWXT_RAMER.f:54:38:

   54 |       real,external :: xmytw_post,esat,tdofesat
      |                                      1
Error: Return type mismatch of function 'esat' at (1) (REAL(8)/REAL(4))
/scratch3/NCEPDEV/nems/Nick.Szapiro/tasks/warnings_upp/ufs-weather-model/UFSATM/upp/sorc/ncep_post.fd/CALWXT_RAMER.f:443:26:

  443 |       b = 26.66082 - alog(es)
      |                          1
Error: In call to 'alog' at (1), type mismatch in argument 'x'; pass 'REAL(4)' to 'REAL(8)'
/scratch3/NCEPDEV/nems/Nick.Szapiro/tasks/warnings_upp/ufs-weather-model/UFSATM/upp/sorc/ncep_post.fd/CALWXT_RAMER.f:54:47:

   54 |       real,external :: xmytw_post,esat,tdofesat
      |                                               1
Error: Return type mismatch of function 'tdofesat' at (1) (REAL(8)/REAL(4))
/scratch3/NCEPDEV/nems/Nick.Szapiro/tasks/warnings_upp/ufs-weather-model/UFSATM/upp/sorc/ncep_post.fd/CALWXT_RAMER.f:54:33:

   54 |       real,external :: xmytw_post,esat,tdofesat
      |                                 1
Error: Return type mismatch of function 'xmytw_post' at (1) (REAL(8)/REAL(4))
/scratch3/NCEPDEV/nems/Nick.Szapiro/tasks/warnings_upp/ufs-weather-model/UFSATM/upp/sorc/ncep_post.fd/CALWXT_RAMER.f:111:38:

  111 |           xxx = max(0.0,min(pq(i,j,l),esat(tq(I,J,L),flag,flg))*rhq(I,J,L))
      |                                      1
Warning: Type mismatch in argument 't' at (1); passed REAL(8) to REAL(4)
/scratch3/NCEPDEV/nems/Nick.Szapiro/tasks/warnings_upp/ufs-weather-model/UFSATM/upp/sorc/ncep_post.fd/CALWXT_RAMER.f:112:16:

  112 |           xxx = tdofesat(xxx,flag,flg)
      |                1
Warning: Type mismatch in argument 'es' at (1); passed REAL(8) to REAL(4)
/scratch3/NCEPDEV/nems/Nick.Szapiro/tasks/warnings_upp/ufs-weather-model/UFSATM/upp/sorc/ncep_post.fd/CALWXT_RAMER.f:113:23:

  113 |           twq(I,J,L) = xmytw_post(tq(I,J,L),xxx,pq(I,J,L))
      |                       1
Warning: Type mismatch in argument 't' at (1); passed REAL(8) to REAL(4)
make[2]: *** [UFSATM/upp/sorc/ncep_post.fd/CMakeFiles/upp.dir/build.make:478: UFSATM/upp/sorc/ncep_post.fd/CMakeFiles/upp.dir/CALWXT_RAMER.f.o] Error 1
```

It looks like https://github.com/NOAA-EMC/UPP/pull/1350 appends ${CMAKE_Fortran_FLAGS} in sorc/ncep_post.fd/CMakeLists.txt
So UPP inherits flags from UFS, including:
```
if(NOT 32BIT)
    set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -real-size 64")
endif()
```
leading to type mismatches.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -128,6 +128,14 @@ if(IFI_FOUND OR INTERNAL_IFI)
   add_definitions(-DUSE_IFI=1)
 endif()
 
+# Enforce 32bit for UPP
+# Intel
+string(REPLACE "-real-size 64" "" CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS}")
+string(REPLACE " -r8 " " " CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS}")
+# GNU
+string(REPLACE "-fdefault-real-8" "" CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS}")
+string(REPLACE "-fdefault-double-8" "" CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS}")
+
 add_subdirectory(sorc)
 
 # Run unit tests.

--- a/tests/logs/rt.log.HERCULES_intel
+++ b/tests/logs/rt.log.HERCULES_intel
@@ -1,65 +1,64 @@
 ===== Start of UPP Regression Testing Log =====
 UPP Hash Tested:
-c38e69e77a4d0b5791b38490bd2eef94d60b72bd
+593fd4c9878cbb0c3d7599b5e9ed39b194309cc9
 
 Submodule hashes:
 -5afd607642ae0fd6a53b137230336386beb47ce6 sorc/libIFI.fd
 -3d35332fe66e3e63a285cc8d96facdf255a33481 sorc/ncep_post.fd/post_gtg.fd
 
-Run directory: /work2/noaa/epic/clyden/regression-tests/upp/hercules/1403_hercules_intel/ci/rundir/upp-HERCULES
+Run directory: /work2/noaa/epic/clyden/regression-tests/upp/hercules/1419_hercules_intel/ci/rundir/upp-HERCULES
 Baseline directory: /work/noaa/epic/role-epic/hercules/UPP
 
-Total runtime: 00h:19m:26s
-Test Date: 20260121 23:09:29
+Total runtime: 00h:09m:51s
+Test Date: 20260123 12:35:25
 Summary Results:
 
-01/22 04:52:30Z -sfs test: your new post executable generates bit-identical sfs.t00z.master.grb2f048 as the develop branch
-01/22 04:52:33Z -aqm test: your new post executable generates bit-identical CMAQ08.tm00 as the develop branch
-01/22 04:52:40Z -gefsv12 test: your new post executable generates bit-identical geaer.t00z.master.grb2f060 as the develop branch
-01/22 04:52:55Z -hafs test: your new post executable generates bit-identical HURPRS09.tm00 as the develop branch
-01/22 04:53:08Z -gefsv13 test: your new post executable generates bit-identical gefs.t00z.master.grb2f009 as the develop branch
-01/22 04:53:22Z -rap test: your new post executable generates bit-identical WRFPRS.GrbF06 as the develop branch
-01/22 04:53:23Z -rap test: your new post executable generates bit-identical WRFNAT.GrbF06 as the develop branch
-01/22 04:53:33Z -nmmb test: your new post executable generates bit-identical BGDAWP03.tm00.Grib2 as the develop branch
-01/22 04:53:33Z -nmmb test: your new post executable generates bit-identical BGRD3D03.tm00.Grib2 as the develop branch
-01/22 04:53:34Z -nmmb test: your new post executable generates bit-identical BGRDSF03.tm00.Grib2 as the develop branch
-01/22 04:53:37Z -mpas test: your new post executable generates bit-identical POSTNAT18.tm00 as the develop branch
-01/22 04:53:37Z -mpas test: your new post executable generates bit-identical POSTPRS18.tm00 as the develop branch
-01/22 04:53:38Z -mpas test: your new post executable generates bit-identical POSTTWO18.tm00 as the develop branch
-01/22 04:54:06Z -3drtma test: your new post executable generates bit-identical WRFNAT.GrbF00 as the develop branch
-01/22 04:54:07Z -3drtma test: your new post executable generates bit-identical WRFTWO.GrbF00 as the develop branch
-01/22 04:54:07Z -3drtma test: your new post executable generates bit-identical WRFPRS.GrbF00 as the develop branch
-01/22 04:54:15Z -hrrr test: your new post executable generates bit-identical WRFTWO.GrbF10 as the develop branch
-01/22 04:54:15Z -hrrr test: your new post executable generates bit-identical WRFPRS.GrbF10 as the develop branch
-01/22 04:54:16Z -hrrr test: your new post executable generates bit-identical WRFNAT.GrbF10 as the develop branch
-01/22 04:54:21Z -mpas_hfip test: your new post executable generates bit-identical NATLEV.GrbF48 as the develop branch
-01/22 04:54:23Z -mpas_hfip test: your new post executable generates bit-identical PRSLEV.GrbF48 as the develop branch
-01/22 04:54:23Z -mpas_hfip test: your new post executable generates bit-identical 2DFLD.GrbF48 as the develop branch
-01/22 04:54:45Z -rrfs_ifi_missing test: your new post executable generates bit-identical IFIFIP18.tm00 as the develop branch
-01/22 04:54:53Z -gcafs test: your new post executable generates bit-identical GFSPRS.GrbF60 as the develop branch
-01/22 04:54:54Z -gcafs test: your new post executable generates bit-identical GFSFLX.GrbF60 as the develop branch
-01/22 05:00:02Z -gfs test: your new post executable generates bit-identical gfs.t00z.master.grb2f006 as the develop branch
-01/22 05:00:04Z -gfs test: your new post executable generates bit-identical gfs.t00z.sfluxgrbf006.grib2 as the develop branch
-01/22 05:00:04Z -gfs test: your new post executable generates bit-identical gfs.t00z.special.grb2f006 as the develop branch
-01/22 05:09:03Z -rrfs test: your new post executable did not generate bit-identical PRSLEV18.tm00 as the develop branch
-01/22 05:09:25Z -rrfs test: your new post executable generates bit-identical NATLEV18.tm00 as the develop branch
-01/22 04:52:54Z -Runtime: sfs_test 00:00:05 -- baseline 00:01:20
-01/22 04:52:54Z -Runtime: aqm_test 00:00:08 -- baseline 00:00:30
-01/22 04:52:55Z -Runtime: gefsv12_test 00:00:15 -- baseline 00:01:00
-01/22 04:53:10Z -Runtime: gefsv13_test 00:00:43 -- baseline 00:02:00
-01/22 04:53:40Z -Runtime: nmmb_test 00:01:09 -- baseline 00:03:00
-01/22 04:53:40Z -Runtime: rap_test 00:00:58 -- baseline 00:02:00
-01/22 04:54:25Z -Runtime: hrrr_test 00:01:51 -- baseline 00:06:00
-01/22 04:54:25Z -Runtime: hafs_test 00:00:30 -- baseline 00:01:00
-01/22 04:54:25Z -Runtime: 3drtma_test 00:01:42 -- baseline 00:03:00
-01/22 04:54:25Z -Runtime: mpas_test 00:01:13 -- baseline 00:02:30
-01/22 04:54:25Z -Runtime: mpas_hfip_test 00:01:59 -- baseline 00:05:00
-01/22 04:54:56Z -Runtime: gcafs_test 00:02:29 -- baseline 00:02:30
-01/22 05:09:28Z -Runtime: rrfs_test 00:17:00 -- baseline 00:12:00
-01/22 05:09:28Z -Runtime: rrfs_ifi_missing 00:02:20 -- baseline 00:04:00
-01/22 05:09:28Z -Runtime: gfs_test 00:07:39 -- baseline 00:25:00
+01/23 18:27:47Z -sfs test: your new post executable generates bit-identical sfs.t00z.master.grb2f048 as the develop branch
+01/23 18:27:50Z -aqm test: your new post executable generates bit-identical CMAQ08.tm00 as the develop branch
+01/23 18:27:56Z -gefsv12 test: your new post executable generates bit-identical geaer.t00z.master.grb2f060 as the develop branch
+01/23 18:28:12Z -hafs test: your new post executable generates bit-identical HURPRS09.tm00 as the develop branch
+01/23 18:28:25Z -gefsv13 test: your new post executable generates bit-identical gefs.t00z.master.grb2f009 as the develop branch
+01/23 18:28:39Z -rap test: your new post executable generates bit-identical WRFPRS.GrbF06 as the develop branch
+01/23 18:28:40Z -rap test: your new post executable generates bit-identical WRFNAT.GrbF06 as the develop branch
+01/23 18:28:49Z -nmmb test: your new post executable generates bit-identical BGDAWP03.tm00.Grib2 as the develop branch
+01/23 18:28:50Z -mpas test: your new post executable generates bit-identical POSTNAT18.tm00 as the develop branch
+01/23 18:28:50Z -nmmb test: your new post executable generates bit-identical BGRD3D03.tm00.Grib2 as the develop branch
+01/23 18:28:50Z -nmmb test: your new post executable generates bit-identical BGRDSF03.tm00.Grib2 as the develop branch
+01/23 18:28:51Z -mpas test: your new post executable generates bit-identical POSTPRS18.tm00 as the develop branch
+01/23 18:28:51Z -mpas test: your new post executable generates bit-identical POSTTWO18.tm00 as the develop branch
+01/23 18:29:21Z -3drtma test: your new post executable generates bit-identical WRFNAT.GrbF00 as the develop branch
+01/23 18:29:22Z -3drtma test: your new post executable generates bit-identical WRFTWO.GrbF00 as the develop branch
+01/23 18:29:22Z -3drtma test: your new post executable generates bit-identical WRFPRS.GrbF00 as the develop branch
+01/23 18:29:30Z -hrrr test: your new post executable generates bit-identical WRFTWO.GrbF10 as the develop branch
+01/23 18:29:31Z -hrrr test: your new post executable generates bit-identical WRFPRS.GrbF10 as the develop branch
+01/23 18:29:32Z -hrrr test: your new post executable generates bit-identical WRFNAT.GrbF10 as the develop branch
+01/23 18:29:33Z -mpas_hfip test: your new post executable generates bit-identical NATLEV.GrbF48 as the develop branch
+01/23 18:29:34Z -mpas_hfip test: your new post executable generates bit-identical PRSLEV.GrbF48 as the develop branch
+01/23 18:29:35Z -mpas_hfip test: your new post executable generates bit-identical 2DFLD.GrbF48 as the develop branch
+01/23 18:29:58Z -rrfs_ifi_missing test: your new post executable generates bit-identical IFIFIP18.tm00 as the develop branch
+01/23 18:30:20Z -gcafs test: your new post executable generates bit-identical GFSPRS.GrbF60 as the develop branch
+01/23 18:30:21Z -gcafs test: your new post executable generates bit-identical GFSFLX.GrbF60 as the develop branch
+01/23 18:34:11Z -rrfs test: your new post executable generates bit-identical PRSLEV18.tm00 as the develop branch
+01/23 18:34:20Z -rrfs test: your new post executable generates bit-identical NATLEV18.tm00 as the develop branch
+01/23 18:35:12Z -gfs test: your new post executable generates bit-identical gfs.t00z.master.grb2f006 as the develop branch
+01/23 18:35:14Z -gfs test: your new post executable generates bit-identical gfs.t00z.sfluxgrbf006.grib2 as the develop branch
+01/23 18:35:14Z -gfs test: your new post executable generates bit-identical gfs.t00z.special.grb2f006 as the develop branch
+01/23 18:28:08Z -Runtime: sfs_test 00:00:06 -- baseline 00:01:20
+01/23 18:28:08Z -Runtime: aqm_test 00:00:09 -- baseline 00:00:30
+01/23 18:28:08Z -Runtime: gefsv12_test 00:00:15 -- baseline 00:01:00
+01/23 18:28:38Z -Runtime: gefsv13_test 00:00:44 -- baseline 00:02:00
+01/23 18:28:53Z -Runtime: nmmb_test 00:01:09 -- baseline 00:03:00
+01/23 18:28:53Z -Runtime: rap_test 00:00:59 -- baseline 00:02:00
+01/23 18:29:38Z -Runtime: hrrr_test 00:01:51 -- baseline 00:06:00
+01/23 18:29:38Z -Runtime: hafs_test 00:00:31 -- baseline 00:01:00
+01/23 18:29:38Z -Runtime: 3drtma_test 00:01:41 -- baseline 00:03:00
+01/23 18:29:38Z -Runtime: mpas_test 00:01:10 -- baseline 00:02:30
+01/23 18:29:38Z -Runtime: mpas_hfip_test 00:01:55 -- baseline 00:05:00
+01/23 18:30:24Z -Runtime: gcafs_test 00:02:40 -- baseline 00:02:30
+01/23 18:34:25Z -Runtime: rrfs_test 00:06:39 -- baseline 00:12:00
+01/23 18:34:25Z -Runtime: rrfs_ifi_missing 00:02:17 -- baseline 00:04:00
+01/23 18:35:25Z -Runtime: gfs_test 00:07:33 -- baseline 00:25:00
 Check tests:
 ['sfs', 'aqm', 'gefsv12', 'gefsv13', 'nmmb', 'rap', 'hrrr', 'hafs', '3drtma', 'mpas', 'mpas_hfip', 'gcafs', 'rrfs', 'rrfs_ifi_missing', 'gfs']
-There are changes in results for case rrfs in /work2/noaa/epic/clyden/regression-tests/upp/hercules/1403_hercules_intel/ci/rundir/upp-HERCULES/rrfs_2025040112/PRSLEV18.tm00
-Refer to .diff files in rundir: /work2/noaa/epic/clyden/regression-tests/upp/hercules/1403_hercules_intel/ci/rundir/upp-HERCULES for details on differences in results for each case.
+No changes in test results detected.
 ===== End of UPP Regression Testing Log =====

--- a/tests/logs/rt.log.ORION_intel
+++ b/tests/logs/rt.log.ORION_intel
@@ -1,65 +1,64 @@
 ===== Start of UPP Regression Testing Log =====
 UPP Hash Tested:
-c38e69e77a4d0b5791b38490bd2eef94d60b72bd
+593fd4c9878cbb0c3d7599b5e9ed39b194309cc9
 
 Submodule hashes:
 -5afd607642ae0fd6a53b137230336386beb47ce6 sorc/libIFI.fd
 -3d35332fe66e3e63a285cc8d96facdf255a33481 sorc/ncep_post.fd/post_gtg.fd
 
-Run directory: /work2/noaa/epic/clyden/regression-tests/upp/orion/1403_orion_intel/ci/rundir/upp-ORION
+Run directory: /work2/noaa/epic/clyden/regression-tests/upp/orion/1419_orion_intel/ci/rundir/upp-ORION
 Baseline directory: /work/noaa/epic/role-epic/orion/UPP
 
-Total runtime: 00h:31m:45s
-Test Date: 20260121 23:21:46
+Total runtime: 00h:15m:29s
+Test Date: 20260123 12:41:02
 Summary Results:
 
-01/22 04:53:20Z -sfs test: your new post executable generates bit-identical sfs.t00z.master.grb2f048 as the develop branch
-01/22 04:53:25Z -aqm test: your new post executable generates bit-identical CMAQ08.tm00 as the develop branch
-01/22 04:53:33Z -gefsv12 test: your new post executable generates bit-identical geaer.t00z.master.grb2f060 as the develop branch
-01/22 04:53:49Z -hafs test: your new post executable generates bit-identical HURPRS09.tm00 as the develop branch
-01/22 04:54:09Z -gefsv13 test: your new post executable generates bit-identical gefs.t00z.master.grb2f009 as the develop branch
-01/22 04:54:41Z -nmmb test: your new post executable generates bit-identical BGDAWP03.tm00.Grib2 as the develop branch
-01/22 04:54:42Z -nmmb test: your new post executable generates bit-identical BGRD3D03.tm00.Grib2 as the develop branch
-01/22 04:54:42Z -nmmb test: your new post executable generates bit-identical BGRDSF03.tm00.Grib2 as the develop branch
-01/22 04:54:57Z -rap test: your new post executable generates bit-identical WRFPRS.GrbF06 as the develop branch
-01/22 04:54:58Z -rap test: your new post executable generates bit-identical WRFNAT.GrbF06 as the develop branch
-01/22 04:55:07Z -mpas test: your new post executable generates bit-identical POSTNAT18.tm00 as the develop branch
-01/22 04:55:08Z -mpas test: your new post executable generates bit-identical POSTPRS18.tm00 as the develop branch
-01/22 04:55:09Z -mpas test: your new post executable generates bit-identical POSTTWO18.tm00 as the develop branch
-01/22 04:55:46Z -rrfs_ifi_missing test: your new post executable generates bit-identical IFIFIP18.tm00 as the develop branch
-01/22 04:56:00Z -gcafs test: your new post executable generates bit-identical GFSPRS.GrbF60 as the develop branch
-01/22 04:56:01Z -gcafs test: your new post executable generates bit-identical GFSFLX.GrbF60 as the develop branch
-01/22 04:56:01Z -3drtma test: your new post executable generates bit-identical WRFNAT.GrbF00 as the develop branch
-01/22 04:56:02Z -3drtma test: your new post executable generates bit-identical WRFTWO.GrbF00 as the develop branch
-01/22 04:56:02Z -3drtma test: your new post executable generates bit-identical WRFPRS.GrbF00 as the develop branch
-01/22 04:56:40Z -mpas_hfip test: your new post executable generates bit-identical NATLEV.GrbF48 as the develop branch
-01/22 04:56:42Z -mpas_hfip test: your new post executable generates bit-identical PRSLEV.GrbF48 as the develop branch
-01/22 04:56:43Z -mpas_hfip test: your new post executable generates bit-identical 2DFLD.GrbF48 as the develop branch
-01/22 04:56:52Z -hrrr test: your new post executable generates bit-identical WRFTWO.GrbF10 as the develop branch
-01/22 04:56:52Z -hrrr test: your new post executable generates bit-identical WRFPRS.GrbF10 as the develop branch
-01/22 04:56:53Z -hrrr test: your new post executable generates bit-identical WRFNAT.GrbF10 as the develop branch
-01/22 05:04:43Z -gfs test: your new post executable generates bit-identical gfs.t00z.master.grb2f006 as the develop branch
-01/22 05:04:45Z -gfs test: your new post executable generates bit-identical gfs.t00z.sfluxgrbf006.grib2 as the develop branch
-01/22 05:04:45Z -gfs test: your new post executable generates bit-identical gfs.t00z.special.grb2f006 as the develop branch
-01/22 05:21:07Z -rrfs test: your new post executable did not generate bit-identical PRSLEV18.tm00 as the develop branch
-01/22 05:21:37Z -rrfs test: your new post executable generates bit-identical NATLEV18.tm00 as the develop branch
-01/22 04:53:42Z -Runtime: sfs_test 00:00:08 -- baseline 00:01:20
-01/22 04:53:42Z -Runtime: aqm_test 00:00:11 -- baseline 00:00:30
-01/22 04:53:42Z -Runtime: gefsv12_test 00:00:19 -- baseline 00:01:00
-01/22 04:54:13Z -Runtime: gefsv13_test 00:00:55 -- baseline 00:02:00
-01/22 04:54:43Z -Runtime: nmmb_test 00:01:28 -- baseline 00:03:00
-01/22 04:55:13Z -Runtime: rap_test 00:01:44 -- baseline 00:02:00
-01/22 04:56:58Z -Runtime: hrrr_test 00:03:39 -- baseline 00:08:00
-01/22 04:56:58Z -Runtime: hafs_test 00:00:36 -- baseline 00:01:00
-01/22 04:56:58Z -Runtime: 3drtma_test 00:02:48 -- baseline 00:03:00
-01/22 04:56:58Z -Runtime: mpas_test 00:01:55 -- baseline 00:02:30
-01/22 04:56:58Z -Runtime: mpas_hfip_test 00:03:29 -- baseline 00:05:00
-01/22 04:56:58Z -Runtime: gcafs_test 00:02:47 -- baseline 00:03:15
-01/22 05:21:46Z -Runtime: rrfs_test 00:28:23 -- baseline 00:12:00
-01/22 05:21:46Z -Runtime: rrfs_ifi_missing 00:02:32 -- baseline 00:04:00
-01/22 05:21:46Z -Runtime: gfs_test 00:11:31 -- baseline 00:26:00
+01/23 18:29:33Z -sfs test: your new post executable generates bit-identical sfs.t00z.master.grb2f048 as the develop branch
+01/23 18:29:37Z -aqm test: your new post executable generates bit-identical CMAQ08.tm00 as the develop branch
+01/23 18:29:46Z -gefsv12 test: your new post executable generates bit-identical geaer.t00z.master.grb2f060 as the develop branch
+01/23 18:30:01Z -hafs test: your new post executable generates bit-identical HURPRS09.tm00 as the develop branch
+01/23 18:30:18Z -gefsv13 test: your new post executable generates bit-identical gefs.t00z.master.grb2f009 as the develop branch
+01/23 18:30:53Z -nmmb test: your new post executable generates bit-identical BGDAWP03.tm00.Grib2 as the develop branch
+01/23 18:30:54Z -nmmb test: your new post executable generates bit-identical BGRD3D03.tm00.Grib2 as the develop branch
+01/23 18:30:54Z -nmmb test: your new post executable generates bit-identical BGRDSF03.tm00.Grib2 as the develop branch
+01/23 18:31:10Z -rap test: your new post executable generates bit-identical WRFPRS.GrbF06 as the develop branch
+01/23 18:31:10Z -rap test: your new post executable generates bit-identical WRFNAT.GrbF06 as the develop branch
+01/23 18:31:17Z -mpas test: your new post executable generates bit-identical POSTNAT18.tm00 as the develop branch
+01/23 18:31:18Z -mpas test: your new post executable generates bit-identical POSTPRS18.tm00 as the develop branch
+01/23 18:31:18Z -mpas test: your new post executable generates bit-identical POSTTWO18.tm00 as the develop branch
+01/23 18:31:58Z -rrfs_ifi_missing test: your new post executable generates bit-identical IFIFIP18.tm00 as the develop branch
+01/23 18:32:10Z -mpas_hfip test: your new post executable generates bit-identical NATLEV.GrbF48 as the develop branch
+01/23 18:32:12Z -mpas_hfip test: your new post executable generates bit-identical PRSLEV.GrbF48 as the develop branch
+01/23 18:32:13Z -mpas_hfip test: your new post executable generates bit-identical 2DFLD.GrbF48 as the develop branch
+01/23 18:32:17Z -3drtma test: your new post executable generates bit-identical WRFNAT.GrbF00 as the develop branch
+01/23 18:32:18Z -3drtma test: your new post executable generates bit-identical WRFTWO.GrbF00 as the develop branch
+01/23 18:32:19Z -3drtma test: your new post executable generates bit-identical WRFPRS.GrbF00 as the develop branch
+01/23 18:32:25Z -gcafs test: your new post executable generates bit-identical GFSPRS.GrbF60 as the develop branch
+01/23 18:32:26Z -gcafs test: your new post executable generates bit-identical GFSFLX.GrbF60 as the develop branch
+01/23 18:33:01Z -hrrr test: your new post executable generates bit-identical WRFTWO.GrbF10 as the develop branch
+01/23 18:33:01Z -hrrr test: your new post executable generates bit-identical WRFPRS.GrbF10 as the develop branch
+01/23 18:33:02Z -hrrr test: your new post executable generates bit-identical WRFNAT.GrbF10 as the develop branch
+01/23 18:39:42Z -rrfs test: your new post executable generates bit-identical PRSLEV18.tm00 as the develop branch
+01/23 18:39:53Z -rrfs test: your new post executable generates bit-identical NATLEV18.tm00 as the develop branch
+01/23 18:40:51Z -gfs test: your new post executable generates bit-identical gfs.t00z.master.grb2f006 as the develop branch
+01/23 18:40:53Z -gfs test: your new post executable generates bit-identical gfs.t00z.sfluxgrbf006.grib2 as the develop branch
+01/23 18:40:53Z -gfs test: your new post executable generates bit-identical gfs.t00z.special.grb2f006 as the develop branch
+01/23 18:29:44Z -Runtime: sfs_test 00:00:08 -- baseline 00:01:20
+01/23 18:29:44Z -Runtime: aqm_test 00:00:12 -- baseline 00:00:30
+01/23 18:29:59Z -Runtime: gefsv12_test 00:00:21 -- baseline 00:01:00
+01/23 18:30:29Z -Runtime: gefsv13_test 00:00:53 -- baseline 00:02:00
+01/23 18:30:59Z -Runtime: nmmb_test 00:01:29 -- baseline 00:03:00
+01/23 18:31:14Z -Runtime: rap_test 00:01:45 -- baseline 00:02:00
+01/23 18:33:14Z -Runtime: hrrr_test 00:03:37 -- baseline 00:08:00
+01/23 18:33:15Z -Runtime: hafs_test 00:00:36 -- baseline 00:01:00
+01/23 18:33:15Z -Runtime: 3drtma_test 00:02:54 -- baseline 00:03:00
+01/23 18:33:15Z -Runtime: mpas_test 00:01:53 -- baseline 00:02:30
+01/23 18:33:15Z -Runtime: mpas_hfip_test 00:03:30 -- baseline 00:05:00
+01/23 18:33:15Z -Runtime: gcafs_test 00:03:01 -- baseline 00:03:15
+01/23 18:40:01Z -Runtime: rrfs_test 00:11:09 -- baseline 00:12:00
+01/23 18:40:01Z -Runtime: rrfs_ifi_missing 00:02:33 -- baseline 00:04:00
+01/23 18:41:01Z -Runtime: gfs_test 00:11:28 -- baseline 00:26:00
 Check tests:
 ['sfs', 'aqm', 'gefsv12', 'gefsv13', 'nmmb', 'rap', 'hrrr', 'hafs', '3drtma', 'mpas', 'mpas_hfip', 'gcafs', 'rrfs', 'rrfs_ifi_missing', 'gfs']
-There are changes in results for case rrfs in /work2/noaa/epic/clyden/regression-tests/upp/orion/1403_orion_intel/ci/rundir/upp-ORION/rrfs_2025040112/PRSLEV18.tm00
-Refer to .diff files in rundir: /work2/noaa/epic/clyden/regression-tests/upp/orion/1403_orion_intel/ci/rundir/upp-ORION for details on differences in results for each case.
+No changes in test results detected.
 ===== End of UPP Regression Testing Log =====

--- a/tests/logs/rt.log.URSA_intel
+++ b/tests/logs/rt.log.URSA_intel
@@ -1,65 +1,64 @@
 ===== Start of UPP Regression Testing Log =====
 UPP Hash Tested:
-c38e69e77a4d0b5791b38490bd2eef94d60b72bd
+593fd4c9878cbb0c3d7599b5e9ed39b194309cc9
 
 Submodule hashes:
 -5afd607642ae0fd6a53b137230336386beb47ce6 sorc/libIFI.fd
 -3d35332fe66e3e63a285cc8d96facdf255a33481 sorc/ncep_post.fd/post_gtg.fd
 
-Run directory: /scratch4/NAGAPE/epic/Chad.Lyden/regression_tests/upp/1403_ursa_intel/ci/rundir/upp-URSA
+Run directory: /scratch4/NAGAPE/epic/Chad.Lyden/regression_tests/upp/1419_ursa_intel/ci/rundir/upp-URSA
 Baseline directory: /scratch4/NAGAPE/epic/role-epic/ursa/UPP/test_suite
 
-Total runtime: 00h:20m:27s
-Test Date: 20260122 05:10:31
+Total runtime: 00h:07m:12s
+Test Date: 20260123 18:32:42
 Summary Results:
 
-01/22 04:52:32Z -aqm test: your new post executable generates bit-identical CMAQ08.tm00 as the develop branch
-01/22 04:52:53Z -hafs test: your new post executable generates bit-identical HURPRS09.tm00 as the develop branch
-01/22 04:53:31Z -mpas test: your new post executable generates bit-identical POSTNAT18.tm00 as the develop branch
-01/22 04:53:32Z -mpas test: your new post executable generates bit-identical POSTPRS18.tm00 as the develop branch
-01/22 04:53:32Z -mpas test: your new post executable generates bit-identical POSTTWO18.tm00 as the develop branch
-01/22 04:53:49Z -3drtma test: your new post executable generates bit-identical WRFNAT.GrbF00 as the develop branch
-01/22 04:53:49Z -3drtma test: your new post executable generates bit-identical WRFTWO.GrbF00 as the develop branch
-01/22 04:53:50Z -3drtma test: your new post executable generates bit-identical WRFPRS.GrbF00 as the develop branch
-01/22 04:55:51Z -mpas_hfip test: your new post executable generates bit-identical NATLEV.GrbF48 as the develop branch
-01/22 04:55:52Z -mpas_hfip test: your new post executable generates bit-identical PRSLEV.GrbF48 as the develop branch
-01/22 04:55:53Z -mpas_hfip test: your new post executable generates bit-identical 2DFLD.GrbF48 as the develop branch
-01/22 04:58:11Z -gfs test: your new post executable generates bit-identical gfs.t00z.master.grb2f006 as the develop branch
-01/22 04:58:12Z -gfs test: your new post executable generates bit-identical gfs.t00z.sfluxgrbf006.grib2 as the develop branch
-01/22 04:58:12Z -gfs test: your new post executable generates bit-identical gfs.t00z.special.grb2f006 as the develop branch
-01/22 04:58:15Z -sfs test: your new post executable generates bit-identical sfs.t00z.master.grb2f048 as the develop branch
-01/22 04:58:23Z -gefsv12 test: your new post executable generates bit-identical geaer.t00z.master.grb2f060 as the develop branch
-01/22 04:58:35Z -nmmb test: your new post executable generates bit-identical BGDAWP03.tm00.Grib2 as the develop branch
-01/22 04:58:36Z -nmmb test: your new post executable generates bit-identical BGRD3D03.tm00.Grib2 as the develop branch
-01/22 04:58:36Z -nmmb test: your new post executable generates bit-identical BGRDSF03.tm00.Grib2 as the develop branch
-01/22 04:58:44Z -gefsv13 test: your new post executable generates bit-identical gefs.t00z.master.grb2f009 as the develop branch
-01/22 04:58:56Z -rap test: your new post executable generates bit-identical WRFPRS.GrbF06 as the develop branch
-01/22 04:58:57Z -rap test: your new post executable generates bit-identical WRFNAT.GrbF06 as the develop branch
-01/22 04:59:23Z -gcafs test: your new post executable generates bit-identical GFSPRS.GrbF60 as the develop branch
-01/22 04:59:24Z -gcafs test: your new post executable generates bit-identical GFSFLX.GrbF60 as the develop branch
-01/22 04:59:36Z -hrrr test: your new post executable generates bit-identical WRFTWO.GrbF10 as the develop branch
-01/22 04:59:37Z -hrrr test: your new post executable generates bit-identical WRFPRS.GrbF10 as the develop branch
-01/22 04:59:38Z -hrrr test: your new post executable generates bit-identical WRFNAT.GrbF10 as the develop branch
-01/22 04:59:57Z -rrfs_ifi_missing test: your new post executable generates bit-identical IFIFIP18.tm00 as the develop branch
-01/22 05:10:24Z -rrfs test: your new post executable did not generate bit-identical PRSLEV18.tm00 as the develop branch
-01/22 05:10:29Z -rrfs test: your new post executable generates bit-identical NATLEV18.tm00 as the develop branch
-01/22 04:58:30Z -Runtime: sfs_test 00:00:04 -- baseline 00:01:20
-01/22 04:58:30Z -Runtime: aqm_test 00:00:04 -- baseline 00:00:30
-01/22 04:58:30Z -Runtime: gefsv12_test 00:00:12 -- baseline 00:02:00
-01/22 04:58:45Z -Runtime: gefsv13_test 00:00:33 -- baseline 00:02:00
-01/22 04:58:45Z -Runtime: nmmb_test 00:00:23 -- baseline 00:02:00
-01/22 04:59:00Z -Runtime: rap_test 00:00:46 -- baseline 00:02:00
-01/22 04:59:45Z -Runtime: hrrr_test 00:01:27 -- baseline 00:03:00
-01/22 04:59:45Z -Runtime: hafs_test 00:00:32 -- baseline 00:01:30
-01/22 04:59:45Z -Runtime: 3drtma_test 00:01:29 -- baseline 00:01:30
-01/22 04:59:45Z -Runtime: mpas_test 00:01:12 -- baseline 00:02:30
-01/22 04:59:45Z -Runtime: mpas_hfip_test 00:03:11 -- baseline 00:05:00
-01/22 04:59:45Z -Runtime: gcafs_test 00:01:13 -- baseline 00:02:00
-01/22 05:10:31Z -Runtime: rrfs_test 00:12:18 -- baseline 00:07:00
-01/22 05:10:31Z -Runtime: rrfs_ifi_missing 00:01:46 -- baseline 00:03:00
-01/22 05:10:31Z -Runtime: gfs_test 00:05:08 -- baseline 00:15:00
+01/23 18:27:25Z -aqm test: your new post executable generates bit-identical CMAQ08.tm00 as the develop branch
+01/23 18:27:41Z -hafs test: your new post executable generates bit-identical HURPRS09.tm00 as the develop branch
+01/23 18:28:04Z -sfs test: your new post executable generates bit-identical sfs.t00z.master.grb2f048 as the develop branch
+01/23 18:28:14Z -gefsv12 test: your new post executable generates bit-identical geaer.t00z.master.grb2f060 as the develop branch
+01/23 18:28:15Z -mpas test: your new post executable generates bit-identical POSTNAT18.tm00 as the develop branch
+01/23 18:28:16Z -mpas test: your new post executable generates bit-identical POSTPRS18.tm00 as the develop branch
+01/23 18:28:17Z -mpas test: your new post executable generates bit-identical POSTTWO18.tm00 as the develop branch
+01/23 18:28:34Z -gefsv13 test: your new post executable generates bit-identical gefs.t00z.master.grb2f009 as the develop branch
+01/23 18:28:39Z -3drtma test: your new post executable generates bit-identical WRFNAT.GrbF00 as the develop branch
+01/23 18:28:39Z -3drtma test: your new post executable generates bit-identical WRFTWO.GrbF00 as the develop branch
+01/23 18:28:40Z -3drtma test: your new post executable generates bit-identical WRFPRS.GrbF00 as the develop branch
+01/23 18:28:46Z -rap test: your new post executable generates bit-identical WRFPRS.GrbF06 as the develop branch
+01/23 18:28:46Z -rap test: your new post executable generates bit-identical WRFNAT.GrbF06 as the develop branch
+01/23 18:28:47Z -nmmb test: your new post executable generates bit-identical BGDAWP03.tm00.Grib2 as the develop branch
+01/23 18:28:47Z -nmmb test: your new post executable generates bit-identical BGRD3D03.tm00.Grib2 as the develop branch
+01/23 18:28:48Z -nmmb test: your new post executable generates bit-identical BGRDSF03.tm00.Grib2 as the develop branch
+01/23 18:28:59Z -gcafs test: your new post executable generates bit-identical GFSPRS.GrbF60 as the develop branch
+01/23 18:29:00Z -gcafs test: your new post executable generates bit-identical GFSFLX.GrbF60 as the develop branch
+01/23 18:29:33Z -rrfs_ifi_missing test: your new post executable generates bit-identical IFIFIP18.tm00 as the develop branch
+01/23 18:29:34Z -hrrr test: your new post executable generates bit-identical WRFTWO.GrbF10 as the develop branch
+01/23 18:29:35Z -hrrr test: your new post executable generates bit-identical WRFPRS.GrbF10 as the develop branch
+01/23 18:29:36Z -hrrr test: your new post executable generates bit-identical WRFNAT.GrbF10 as the develop branch
+01/23 18:30:28Z -mpas_hfip test: your new post executable generates bit-identical NATLEV.GrbF48 as the develop branch
+01/23 18:30:30Z -mpas_hfip test: your new post executable generates bit-identical PRSLEV.GrbF48 as the develop branch
+01/23 18:30:31Z -mpas_hfip test: your new post executable generates bit-identical 2DFLD.GrbF48 as the develop branch
+01/23 18:32:16Z -rrfs test: your new post executable generates bit-identical PRSLEV18.tm00 as the develop branch
+01/23 18:32:34Z -rrfs test: your new post executable generates bit-identical NATLEV18.tm00 as the develop branch
+01/23 18:32:38Z -gfs test: your new post executable generates bit-identical gfs.t00z.master.grb2f006 as the develop branch
+01/23 18:32:39Z -gfs test: your new post executable generates bit-identical gfs.t00z.sfluxgrbf006.grib2 as the develop branch
+01/23 18:32:39Z -gfs test: your new post executable generates bit-identical gfs.t00z.special.grb2f006 as the develop branch
+01/23 18:28:11Z -Runtime: sfs_test 00:00:04 -- baseline 00:01:20
+01/23 18:28:11Z -Runtime: aqm_test 00:00:05 -- baseline 00:00:30
+01/23 18:28:26Z -Runtime: gefsv12_test 00:00:12 -- baseline 00:02:00
+01/23 18:28:41Z -Runtime: gefsv13_test 00:00:32 -- baseline 00:02:00
+01/23 18:28:56Z -Runtime: nmmb_test 00:00:23 -- baseline 00:02:00
+01/23 18:28:56Z -Runtime: rap_test 00:00:44 -- baseline 00:02:00
+01/23 18:29:42Z -Runtime: hrrr_test 00:01:27 -- baseline 00:03:00
+01/23 18:29:42Z -Runtime: hafs_test 00:00:28 -- baseline 00:01:30
+01/23 18:29:42Z -Runtime: 3drtma_test 00:01:27 -- baseline 00:01:30
+01/23 18:29:42Z -Runtime: mpas_test 00:01:04 -- baseline 00:02:30
+01/23 18:30:42Z -Runtime: mpas_hfip_test 00:03:18 -- baseline 00:05:00
+01/23 18:30:42Z -Runtime: gcafs_test 00:01:13 -- baseline 00:02:00
+01/23 18:32:42Z -Runtime: rrfs_test 00:04:52 -- baseline 00:07:00
+01/23 18:32:42Z -Runtime: rrfs_ifi_missing 00:01:46 -- baseline 00:03:00
+01/23 18:32:42Z -Runtime: gfs_test 00:05:10 -- baseline 00:15:00
 Check tests:
 ['sfs', 'aqm', 'gefsv12', 'gefsv13', 'nmmb', 'rap', 'hrrr', 'hafs', '3drtma', 'mpas', 'mpas_hfip', 'gcafs', 'rrfs', 'rrfs_ifi_missing', 'gfs']
-There are changes in results for case rrfs in /scratch4/NAGAPE/epic/Chad.Lyden/regression_tests/upp/1403_ursa_intel/ci/rundir/upp-URSA/rrfs_2025040112/PRSLEV18.tm00
-Refer to .diff files in rundir: /scratch4/NAGAPE/epic/Chad.Lyden/regression_tests/upp/1403_ursa_intel/ci/rundir/upp-URSA for details on differences in results for each case.
+No changes in test results detected.
 ===== End of UPP Regression Testing Log =====

--- a/tests/logs/rt.log.URSA_intelllvm
+++ b/tests/logs/rt.log.URSA_intelllvm
@@ -1,65 +1,64 @@
 ===== Start of UPP Regression Testing Log =====
 UPP Hash Tested:
-c38e69e77a4d0b5791b38490bd2eef94d60b72bd
+593fd4c9878cbb0c3d7599b5e9ed39b194309cc9
 
 Submodule hashes:
 -5afd607642ae0fd6a53b137230336386beb47ce6 sorc/libIFI.fd
 -3d35332fe66e3e63a285cc8d96facdf255a33481 sorc/ncep_post.fd/post_gtg.fd
 
-Run directory: /scratch4/NAGAPE/epic/Chad.Lyden/regression_tests/upp/1403_ursa_intelllvm/ci/rundir/upp-URSA
+Run directory: /scratch4/NAGAPE/epic/Chad.Lyden/regression_tests/upp/1419_ursa_intelllvm/ci/rundir/upp-URSA
 Baseline directory: /scratch4/NAGAPE/epic/role-epic/ursa/UPP/test_suite
 
-Total runtime: 00h:12m:54s
-Test Date: 20260122 05:02:57
+Total runtime: 00h:06m:39s
+Test Date: 20260123 18:32:10
 Summary Results:
 
-01/22 04:51:32Z -hafs test: your new post executable generates bit-identical HURPRS09.tm00 as the develop branch
-01/22 04:52:01Z -mpas test: your new post executable generates bit-identical POSTNAT18.tm00 as the develop branch
-01/22 04:52:02Z -mpas test: your new post executable generates bit-identical POSTPRS18.tm00 as the develop branch
-01/22 04:52:02Z -mpas test: your new post executable generates bit-identical POSTTWO18.tm00 as the develop branch
-01/22 04:52:17Z -3drtma test: your new post executable generates bit-identical WRFNAT.GrbF00 as the develop branch
-01/22 04:52:17Z -3drtma test: your new post executable generates bit-identical WRFTWO.GrbF00 as the develop branch
-01/22 04:52:18Z -3drtma test: your new post executable generates bit-identical WRFPRS.GrbF00 as the develop branch
-01/22 04:52:27Z -aqm test: your new post executable generates bit-identical CMAQ08.tm00 as the develop branch
-01/22 04:52:41Z -rrfs_ifi_missing test: your new post executable generates bit-identical IFIFIP18.tm00 as the develop branch
-01/22 04:53:00Z -sfs test: your new post executable generates bit-identical sfs.t00z.master.grb2f048 as the develop branch
-01/22 04:53:13Z -gefsv12 test: your new post executable generates bit-identical geaer.t00z.master.grb2f060 as the develop branch
-01/22 04:54:03Z -mpas_hfip test: your new post executable generates bit-identical NATLEV.GrbF48 as the develop branch
-01/22 04:54:05Z -mpas_hfip test: your new post executable generates bit-identical PRSLEV.GrbF48 as the develop branch
-01/22 04:54:06Z -mpas_hfip test: your new post executable generates bit-identical 2DFLD.GrbF48 as the develop branch
-01/22 04:54:25Z -gcafs test: your new post executable generates bit-identical GFSPRS.GrbF60 as the develop branch
-01/22 04:54:26Z -gcafs test: your new post executable generates bit-identical GFSFLX.GrbF60 as the develop branch
-01/22 04:57:44Z -gfs test: your new post executable generates bit-identical gfs.t00z.master.grb2f006 as the develop branch
-01/22 04:57:45Z -gfs test: your new post executable generates bit-identical gfs.t00z.sfluxgrbf006.grib2 as the develop branch
-01/22 04:57:45Z -gfs test: your new post executable generates bit-identical gfs.t00z.special.grb2f006 as the develop branch
-01/22 04:58:30Z -nmmb test: your new post executable generates bit-identical BGDAWP03.tm00.Grib2 as the develop branch
-01/22 04:58:31Z -nmmb test: your new post executable generates bit-identical BGRD3D03.tm00.Grib2 as the develop branch
-01/22 04:58:31Z -nmmb test: your new post executable generates bit-identical BGRDSF03.tm00.Grib2 as the develop branch
-01/22 04:58:35Z -gefsv13 test: your new post executable generates bit-identical gefs.t00z.master.grb2f009 as the develop branch
-01/22 04:58:50Z -rap test: your new post executable generates bit-identical WRFPRS.GrbF06 as the develop branch
-01/22 04:58:50Z -rap test: your new post executable generates bit-identical WRFNAT.GrbF06 as the develop branch
-01/22 04:59:15Z -hrrr test: your new post executable generates bit-identical WRFTWO.GrbF10 as the develop branch
-01/22 04:59:16Z -hrrr test: your new post executable generates bit-identical WRFPRS.GrbF10 as the develop branch
-01/22 04:59:17Z -hrrr test: your new post executable generates bit-identical WRFNAT.GrbF10 as the develop branch
-01/22 05:02:50Z -rrfs test: your new post executable did not generate bit-identical PRSLEV18.tm00 as the develop branch
-01/22 05:02:55Z -rrfs test: your new post executable generates bit-identical NATLEV18.tm00 as the develop branch
-01/22 04:53:11Z -Runtime: sfs_test 00:00:06 -- baseline 00:01:20
-01/22 04:53:11Z -Runtime: aqm_test 00:00:06 -- baseline 00:00:30
-01/22 04:53:26Z -Runtime: gefsv12_test 00:00:13 -- baseline 00:02:00
-01/22 04:58:41Z -Runtime: gefsv13_test 00:00:34 -- baseline 00:02:00
-01/22 04:58:41Z -Runtime: nmmb_test 00:00:20 -- baseline 00:01:30
-01/22 04:58:56Z -Runtime: rap_test 00:00:40 -- baseline 00:02:00
-01/22 04:59:26Z -Runtime: hrrr_test 00:01:06 -- baseline 00:02:30
-01/22 04:59:26Z -Runtime: hafs_test 00:00:35 -- baseline 00:01:30
-01/22 04:59:26Z -Runtime: 3drtma_test 00:01:21 -- baseline 00:01:30
-01/22 04:59:26Z -Runtime: mpas_test 00:01:05 -- baseline 00:02:30
-01/22 04:59:26Z -Runtime: mpas_hfip_test 00:03:09 -- baseline 00:05:00
-01/22 04:59:27Z -Runtime: gcafs_test 00:01:11 -- baseline 00:02:30
-01/22 05:02:57Z -Runtime: rrfs_test 00:11:58 -- baseline 00:06:00
-01/22 05:02:57Z -Runtime: rrfs_ifi_missing 00:01:44 -- baseline 00:03:00
-01/22 05:02:57Z -Runtime: gfs_test 00:04:53 -- baseline 00:15:00
+01/23 18:26:53Z -hafs test: your new post executable generates bit-identical HURPRS09.tm00 as the develop branch
+01/23 18:27:06Z -aqm test: your new post executable generates bit-identical CMAQ08.tm00 as the develop branch
+01/23 18:27:14Z -sfs test: your new post executable generates bit-identical sfs.t00z.master.grb2f048 as the develop branch
+01/23 18:27:38Z -mpas test: your new post executable generates bit-identical POSTNAT18.tm00 as the develop branch
+01/23 18:27:39Z -mpas test: your new post executable generates bit-identical POSTPRS18.tm00 as the develop branch
+01/23 18:27:39Z -mpas test: your new post executable generates bit-identical POSTTWO18.tm00 as the develop branch
+01/23 18:27:44Z -gefsv12 test: your new post executable generates bit-identical geaer.t00z.master.grb2f060 as the develop branch
+01/23 18:27:44Z -3drtma test: your new post executable generates bit-identical WRFNAT.GrbF00 as the develop branch
+01/23 18:27:44Z -3drtma test: your new post executable generates bit-identical WRFTWO.GrbF00 as the develop branch
+01/23 18:27:45Z -3drtma test: your new post executable generates bit-identical WRFPRS.GrbF00 as the develop branch
+01/23 18:28:07Z -gefsv13 test: your new post executable generates bit-identical gefs.t00z.master.grb2f009 as the develop branch
+01/23 18:28:38Z -rap test: your new post executable generates bit-identical WRFPRS.GrbF06 as the develop branch
+01/23 18:28:38Z -rap test: your new post executable generates bit-identical WRFNAT.GrbF06 as the develop branch
+01/23 18:28:41Z -nmmb test: your new post executable generates bit-identical BGDAWP03.tm00.Grib2 as the develop branch
+01/23 18:28:41Z -nmmb test: your new post executable generates bit-identical BGRD3D03.tm00.Grib2 as the develop branch
+01/23 18:28:42Z -nmmb test: your new post executable generates bit-identical BGRDSF03.tm00.Grib2 as the develop branch
+01/23 18:28:54Z -gcafs test: your new post executable generates bit-identical GFSPRS.GrbF60 as the develop branch
+01/23 18:28:55Z -gcafs test: your new post executable generates bit-identical GFSFLX.GrbF60 as the develop branch
+01/23 18:29:02Z -rrfs_ifi_missing test: your new post executable generates bit-identical IFIFIP18.tm00 as the develop branch
+01/23 18:29:06Z -hrrr test: your new post executable generates bit-identical WRFTWO.GrbF10 as the develop branch
+01/23 18:29:07Z -hrrr test: your new post executable generates bit-identical WRFPRS.GrbF10 as the develop branch
+01/23 18:29:08Z -hrrr test: your new post executable generates bit-identical WRFNAT.GrbF10 as the develop branch
+01/23 18:29:46Z -mpas_hfip test: your new post executable generates bit-identical NATLEV.GrbF48 as the develop branch
+01/23 18:29:48Z -mpas_hfip test: your new post executable generates bit-identical PRSLEV.GrbF48 as the develop branch
+01/23 18:29:49Z -mpas_hfip test: your new post executable generates bit-identical 2DFLD.GrbF48 as the develop branch
+01/23 18:31:29Z -gfs test: your new post executable generates bit-identical gfs.t00z.master.grb2f006 as the develop branch
+01/23 18:31:30Z -gfs test: your new post executable generates bit-identical gfs.t00z.sfluxgrbf006.grib2 as the develop branch
+01/23 18:31:30Z -gfs test: your new post executable generates bit-identical gfs.t00z.special.grb2f006 as the develop branch
+01/23 18:31:46Z -rrfs test: your new post executable generates bit-identical PRSLEV18.tm00 as the develop branch
+01/23 18:32:05Z -rrfs test: your new post executable generates bit-identical NATLEV18.tm00 as the develop branch
+01/23 18:27:24Z -Runtime: sfs_test 00:00:06 -- baseline 00:01:20
+01/23 18:27:24Z -Runtime: aqm_test 00:00:05 -- baseline 00:00:30
+01/23 18:27:54Z -Runtime: gefsv12_test 00:00:12 -- baseline 00:02:00
+01/23 18:28:09Z -Runtime: gefsv13_test 00:00:33 -- baseline 00:02:00
+01/23 18:28:54Z -Runtime: nmmb_test 00:00:20 -- baseline 00:01:30
+01/23 18:28:54Z -Runtime: rap_test 00:00:39 -- baseline 00:02:00
+01/23 18:29:09Z -Runtime: hrrr_test 00:01:08 -- baseline 00:02:30
+01/23 18:29:09Z -Runtime: hafs_test 00:00:27 -- baseline 00:01:30
+01/23 18:29:09Z -Runtime: 3drtma_test 00:01:19 -- baseline 00:01:30
+01/23 18:29:09Z -Runtime: mpas_test 00:01:13 -- baseline 00:02:30
+01/23 18:29:54Z -Runtime: mpas_hfip_test 00:03:07 -- baseline 00:05:00
+01/23 18:29:54Z -Runtime: gcafs_test 00:01:10 -- baseline 00:02:30
+01/23 18:32:10Z -Runtime: rrfs_test 00:04:48 -- baseline 00:06:00
+01/23 18:32:10Z -Runtime: rrfs_ifi_missing 00:01:45 -- baseline 00:03:00
+01/23 18:32:10Z -Runtime: gfs_test 00:04:48 -- baseline 00:15:00
 Check tests:
 ['sfs', 'aqm', 'gefsv12', 'gefsv13', 'nmmb', 'rap', 'hrrr', 'hafs', '3drtma', 'mpas', 'mpas_hfip', 'gcafs', 'rrfs', 'rrfs_ifi_missing', 'gfs']
-There are changes in results for case rrfs in /scratch4/NAGAPE/epic/Chad.Lyden/regression_tests/upp/1403_ursa_intelllvm/ci/rundir/upp-URSA/rrfs_2025040112/PRSLEV18.tm00
-Refer to .diff files in rundir: /scratch4/NAGAPE/epic/Chad.Lyden/regression_tests/upp/1403_ursa_intelllvm/ci/rundir/upp-URSA for details on differences in results for each case.
+No changes in test results detected.
 ===== End of UPP Regression Testing Log =====

--- a/tests/logs/rt.log.WCOSS2_intel
+++ b/tests/logs/rt.log.WCOSS2_intel
@@ -1,65 +1,71 @@
 ===== Start of UPP Regression Testing Log =====
 UPP Hash Tested:
-51eb60a66d4e8a2f0cd572e1d9a889ffc726c4ad
+9edac666a247a6de4c601ca2bdf8d5aad465bddb
 
 Submodule hashes:
--5afd607642ae0fd6a53b137230336386beb47ce6 sorc/libIFI.fd
--3d35332fe66e3e63a285cc8d96facdf255a33481 sorc/ncep_post.fd/post_gtg.fd
+ 5afd607642ae0fd6a53b137230336386beb47ce6 sorc/libIFI.fd (before-3km-fixes-54-g5afd607)
+ 3d35332fe66e3e63a285cc8d96facdf255a33481 sorc/ncep_post.fd/post_gtg.fd (ncep_post_gtg.v1.0.6-64-g3d35332)
 
-Run directory: /lfs/h2/emc/ptmp/wen.meng/upp-WCOSS2
+Run directory: /lfs/h2/emc/ptmp/Benjamin.Blake/upp-WCOSS2
 Baseline directory: /lfs/h2/emc/vpppg/noscrub/wen.meng/test_suite
 
-Total runtime: 00h:24m:23s
-Test Date: 20260121 14:01:32
+Total runtime: 00h:17m:28s
+Test Date: 20260123 13:58:55
 Summary Results:
 
-01/21 13:40:27Z -sfs test: your new post executable generates bit-identical sfs.t00z.master.grb2f048 as the develop branch
-01/21 13:40:45Z -aqm test: your new post executable generates bit-identical CMAQ08.tm00 as the develop branch
-01/21 13:40:53Z -gefsv12 test: your new post executable generates bit-identical geaer.t00z.master.grb2f060 as the develop branch
-01/21 13:41:17Z -hafs test: your new post executable generates bit-identical HURPRS09.tm00 as the develop branch
-01/21 13:41:38Z -rap test: your new post executable generates bit-identical WRFPRS.GrbF06 as the develop branch
-01/21 13:41:39Z -gefsv13 test: your new post executable generates bit-identical gefs.t00z.master.grb2f009 as the develop branch
-01/21 13:41:39Z -rap test: your new post executable generates bit-identical WRFNAT.GrbF06 as the develop branch
-01/21 13:42:07Z -mpas test: your new post executable generates bit-identical POSTNAT18.tm00 as the develop branch
-01/21 13:42:08Z -nmmb test: your new post executable generates bit-identical BGDAWP03.tm00.Grib2 as the develop branch
-01/21 13:42:10Z -nmmb test: your new post executable generates bit-identical BGRD3D03.tm00.Grib2 as the develop branch
-01/21 13:42:11Z -mpas test: your new post executable generates bit-identical POSTPRS18.tm00 as the develop branch
-01/21 13:42:11Z -nmmb test: your new post executable generates bit-identical BGRDSF03.tm00.Grib2 as the develop branch
-01/21 13:42:12Z -mpas test: your new post executable generates bit-identical POSTTWO18.tm00 as the develop branch
-01/21 13:42:37Z -3drtma test: your new post executable generates bit-identical WRFNAT.GrbF00 as the develop branch
-01/21 13:42:39Z -3drtma test: your new post executable generates bit-identical WRFTWO.GrbF00 as the develop branch
-01/21 13:42:41Z -3drtma test: your new post executable generates bit-identical WRFPRS.GrbF00 as the develop branch
-01/21 13:43:10Z -hrrr test: your new post executable generates bit-identical WRFTWO.GrbF10 as the develop branch
-01/21 13:43:12Z -hrrr test: your new post executable generates bit-identical WRFPRS.GrbF10 as the develop branch
-01/21 13:43:14Z -hrrr test: your new post executable generates bit-identical WRFNAT.GrbF10 as the develop branch
-01/21 13:44:00Z -gcafs test: your new post executable generates bit-identical GFSPRS.GrbF60 as the develop branch
-01/21 13:44:02Z -gcafs test: your new post executable generates bit-identical GFSFLX.GrbF60 as the develop branch
-01/21 13:44:28Z -mpas_hfip test: your new post executable generates bit-identical NATLEV.GrbF48 as the develop branch
-01/21 13:44:34Z -mpas_hfip test: your new post executable generates bit-identical PRSLEV.GrbF48 as the develop branch
-01/21 13:44:36Z -mpas_hfip test: your new post executable generates bit-identical 2DFLD.GrbF48 as the develop branch
-01/21 13:45:17Z -rrfs_ifi_missing test: your new post executable generates bit-identical IFIFIP18.tm00 as the develop branch
-01/21 13:51:12Z -gfs test: your new post executable generates bit-identical gfs.t00z.master.grb2f006 as the develop branch
-01/21 13:51:14Z -gfs test: your new post executable generates bit-identical gfs.t00z.sfluxgrbf006.grib2 as the develop branch
-01/21 13:51:14Z -gfs test: your new post executable generates bit-identical gfs.t00z.special.grb2f006 as the develop branch
-01/21 14:00:33Z -rrfs test: your new post executable did not generate bit-identical PRSLEV18.tm00 as the develop branch
-01/21 14:01:00Z -rrfs test: your new post executable generates bit-identical NATLEV18.tm00 as the develop branch
-01/21 13:40:50Z -Runtime: sfs_test 00:00:34 -- baseline 00:00:50
-01/21 13:41:11Z -Runtime: aqm_test 00:00:55 -- baseline 00:01:00
-01/21 13:41:15Z -Runtime: gefsv12_test 00:01:04 -- baseline 00:01:20
-01/21 13:42:08Z -Runtime: gefsv13_test 00:01:50 -- baseline 00:02:00
-01/21 13:42:28Z -Runtime: nmmb_test 00:02:22 -- baseline 00:02:20
-01/21 13:42:33Z -Runtime: rap_test 00:01:51 -- baseline 00:02:00
-01/21 13:43:41Z -Runtime: hrrr_test 00:03:26 -- baseline 00:03:40
-01/21 13:43:46Z -Runtime: hafs_test 00:01:29 -- baseline 00:02:00
-01/21 13:43:50Z -Runtime: 3drtma_test 00:02:53 -- baseline 00:03:00
-01/21 13:43:54Z -Runtime: mpas_test 00:02:24 -- baseline 00:02:40
-01/21 13:45:03Z -Runtime: mpas_hfip.test 00:04:49 -- baseline 00:05:00
-01/21 13:45:07Z -Runtime: gcafs_test 00:04:17 -- baseline 00:05:30
-01/21 14:01:23Z -Runtime: rrfs_test 00:21:17 -- baseline 00:10:00
-01/21 14:01:27Z -Runtime: rrfs_ifi_mis 00:05:30 -- baseline 00:08:00
-01/21 14:01:32Z -Runtime: gfs_test 00:11:27 -- baseline 00:15:00
+01/23 13:47:48Z -sfs test: your new post executable generates bit-identical sfs.t00z.master.grb2f048 as the develop branch
+01/23 13:48:00Z -aqm test: your new post executable generates bit-identical CMAQ08.tm00 as the develop branch
+01/23 13:48:09Z -gefsv12 test: your new post executable generates bit-identical geaer.t00z.master.grb2f060 as the develop branch
+01/23 13:48:30Z -hafs test: your new post executable generates bit-identical HURPRS09.tm00 as the develop branch
+01/23 13:48:38Z -hrrr_ifi test: your new post executable generates bit-identical IFIFIP.GrbF10 as the develop branch
+01/23 13:48:49Z -dafs test: your new post executable generates bit-identical AVIATION.GrbF08 as the develop branch
+01/23 13:48:49Z -dafs test: your new post executable generates bit-identical IFIFIP.GrbF08 as the develop branch
+01/23 13:48:53Z -nmmb test: your new post executable generates bit-identical BGDAWP03.tm00.Grib2 as the develop branch
+01/23 13:48:53Z -rap test: your new post executable generates bit-identical WRFPRS.GrbF06 as the develop branch
+01/23 13:48:54Z -nmmb test: your new post executable generates bit-identical BGRD3D03.tm00.Grib2 as the develop branch
+01/23 13:48:54Z -rap test: your new post executable generates bit-identical WRFNAT.GrbF06 as the develop branch
+01/23 13:48:54Z -nmmb test: your new post executable generates bit-identical BGRDSF03.tm00.Grib2 as the develop branch
+01/23 13:48:56Z -gefsv13 test: your new post executable generates bit-identical gefs.t00z.master.grb2f009 as the develop branch
+01/23 13:49:15Z -mpas test: your new post executable generates bit-identical POSTNAT18.tm00 as the develop branch
+01/23 13:49:17Z -mpas test: your new post executable generates bit-identical POSTPRS18.tm00 as the develop branch
+01/23 13:49:18Z -mpas test: your new post executable generates bit-identical POSTTWO18.tm00 as the develop branch
+01/23 13:49:43Z -3drtma test: your new post executable generates bit-identical WRFNAT.GrbF00 as the develop branch
+01/23 13:49:43Z -3drtma test: your new post executable generates bit-identical WRFTWO.GrbF00 as the develop branch
+01/23 13:49:45Z -3drtma test: your new post executable generates bit-identical WRFPRS.GrbF00 as the develop branch
+01/23 13:50:20Z -hrrr test: your new post executable generates bit-identical WRFTWO.GrbF10 as the develop branch
+01/23 13:50:21Z -hrrr test: your new post executable generates bit-identical WRFPRS.GrbF10 as the develop branch
+01/23 13:50:23Z -hrrr test: your new post executable generates bit-identical WRFNAT.GrbF10 as the develop branch
+01/23 13:51:00Z -gcafs test: your new post executable generates bit-identical GFSPRS.GrbF60 as the develop branch
+01/23 13:51:05Z -gcafs test: your new post executable generates bit-identical GFSFLX.GrbF60 as the develop branch
+01/23 13:51:36Z -mpas_hfip test: your new post executable generates bit-identical NATLEV.GrbF48 as the develop branch
+01/23 13:51:42Z -mpas_hfip test: your new post executable generates bit-identical PRSLEV.GrbF48 as the develop branch
+01/23 13:51:45Z -mpas_hfip test: your new post executable generates bit-identical 2DFLD.GrbF48 as the develop branch
+01/23 13:52:54Z -rrfs_ifi_missing test: your new post executable generates bit-identical IFIFIP18.tm00 as the develop branch
+01/23 13:53:50Z -rrfs_ifi test: your new post executable generates bit-identical IFIFIP18.tm00 as the develop branch
+01/23 13:56:23Z -rrfs test: your new post executable generates bit-identical PRSLEV18.tm00 as the develop branch
+01/23 13:56:44Z -rrfs test: your new post executable generates bit-identical NATLEV18.tm00 as the develop branch
+01/23 13:58:15Z -gfs test: your new post executable generates bit-identical gfs.t00z.master.grb2f006 as the develop branch
+01/23 13:58:16Z -gfs test: your new post executable generates bit-identical gfs.t00z.sfluxgrbf006.grib2 as the develop branch
+01/23 13:58:16Z -gfs test: your new post executable generates bit-identical gfs.t00z.special.grb2f006 as the develop branch
+01/23 13:48:14Z -Runtime: sfs_test 00:00:47 -- baseline 00:00:50
+01/23 13:48:18Z -Runtime: aqm_test 00:01:03 -- baseline 00:01:00
+01/23 13:48:39Z -Runtime: gefsv12_test 00:01:12 -- baseline 00:01:20
+01/23 13:49:15Z -Runtime: gefsv13_test 00:01:59 -- baseline 00:02:00
+01/23 13:49:20Z -Runtime: nmmb_test 00:01:57 -- baseline 00:02:20
+01/23 13:49:24Z -Runtime: rap_test 00:01:57 -- baseline 00:02:00
+01/23 13:50:49Z -Runtime: hrrr_test 00:03:28 -- baseline 00:03:40
+01/23 13:50:53Z -Runtime: hafs_test 00:01:33 -- baseline 00:02:00
+01/23 13:50:57Z -Runtime: 3drtma_test 00:02:49 -- baseline 00:03:00
+01/23 13:51:02Z -Runtime: mpas_test 00:02:21 -- baseline 00:02:40
+01/23 13:52:10Z -Runtime: mpas_hfip.test 00:04:50 -- baseline 00:05:00
+01/23 13:52:15Z -Runtime: gcafs_test 00:04:09 -- baseline 00:05:30
+01/23 13:57:12Z -Runtime: rrfs_test 00:09:55 -- baseline 00:10:00
+01/23 13:57:17Z -Runtime: rrfs_ifi_mis 00:06:01 -- baseline 00:08:00
+01/23 13:58:42Z -Runtime: gfs_test 00:11:22 -- baseline 00:15:00
+01/23 13:58:46Z -Runtime: hrrr_ifi_test 00:01:40 -- baseline 00:02:00
+01/23 13:58:50Z -Runtime: rrfs_ifi_test 00:06:54 -- baseline 00:08:10
+01/23 13:58:55Z -Runtime: dafs_test 00:01:51 -- baseline 00:02:00
 Check tests:
-['sfs', 'aqm', 'gefsv12', 'gefsv13', 'nmmb', 'rap', 'hrrr', 'hafs', '3drtma', 'mpas', 'mpas_hfip', 'gcafs', 'rrfs', 'rrfs_ifi_missing', 'gfs']
-There are changes in results for case rrfs in /lfs/h2/emc/ptmp/wen.meng/upp-WCOSS2/rrfs_2025040112/PRSLEV18.tm00
-Refer to .diff files in rundir: /lfs/h2/emc/ptmp/wen.meng/upp-WCOSS2 for details on differences in results for each case.
+['sfs', 'aqm', 'gefsv12', 'gefsv13', 'nmmb', 'rap', 'hrrr', 'hafs', '3drtma', 'mpas', 'mpas_hfip', 'gcafs', 'rrfs', 'rrfs_ifi_missing', 'gfs', 'hrrr_ifi', 'rrfs_ifi', 'dafs']
+No changes in test results detected.
 ===== End of UPP Regression Testing Log =====

